dao/user_dao_test.go

```diff
@@ -0,0 +1,46 @@
+package dao
+
+import (
+	"testing"
+
+	"github.com/RedHatInsights/sources-api-go/internal/testutils"
+	"github.com/RedHatInsights/sources-api-go/internal/testutils/fixtures"
+	"github.com/RedHatInsights/sources-api-go/model"
+)
+
+// TestGetOrCreateTenantIDEbsNumberCreate tests that when the EBS account number is not found, a new tenant is created.
+func TestFindOrCreateUser(t *testing.T) {
```

Comment:
here is not updated commend above TestFindOrCreateUser()

Reply:
Comment removed.